__test__/class.test.js

```diff
@@ -0,0 +1,33 @@
+import ToDo from '../modules/class.js';
```

Comment:
- [ ] [OPTIONAL] I would suggest adding more tests for your app, you can make tests for checkboxes, and clear completed tasks,...